src/vaults/yields.utils.ts

```diff
@@ -347,11 +318,6 @@ export function createYieldSource(
 }
 
 export async function loadVaultEventPerformances(chain: Chain, vault: VaultDefinitionModel): Promise<YieldSource[]> {
-  const incompatibleNetworks = new Set<Network>([Network.BinanceSmartChain, Network.Polygon, Network.Arbitrum]);
```

Comment:
what happened here

Reply:
these networks who do not support the historic queries will anyway throw an error and so bc this vault is supported in the try / catch from the caller fn removal does not impact the end result of the fn.

also, it is possible in the future or if utilizing archival RPC that this is not true either - at the time it was written though we weren't and arbitrum as a specific example did not have any custom rpc or any archival capabilities on its launch.